CMakeLists.txt

```diff
@@ -214,9 +214,7 @@ find_package(PythonInterp REQUIRED)
 
 if (WITH_PYTHON)
   find_package(Cython 0.23 REQUIRED)
-  execute_process(COMMAND ${PYTHON_EXECUTABLE}
-                -c "import distutils.sysconfig as cg; print(cg.get_python_inc())"
-                OUTPUT_VARIABLE PYTHON_INCLUDE_DIRS OUTPUT_STRIP_TRAILING_WHITESPACE)
+  find_package(PythonLibs 2.7 REQUIRED)
```

Comment:
Please revert this change. The old version is necessary so `PythonInterpreter` and `PythonLibs` actually are the same Python.